app/models/seed_data.rb

```diff
@@ -125,10 +125,12 @@ def initialize
       {
         id: 202,
         placement_need_id: 202,
+        placement_types: "long_term",
```

Comment:
we can only pass a single placement_type from the placement_need form, I assume we are differentiating from this behaviour in the filter form?

Reply:
In the seed data se only setup an "initial" data for testing, so same as in the actual journey, the `Shortlist` is created on the back of `PlacementNeed`. The reason for having the field here again is to be able to persist the "filter" independently of the `PlacementNeed`. 

When you use the form, you can provide as many placement types as possible and they are stored comma separated in this field. Again, it is not ideal, but it's only to power the filter and so I think a bit of de-normalisation is ok. Should this be used for more than that, we should possibly change it.